zk-tests/src/Cheatcodes.t.sol

```diff
@@ -68,13 +81,19 @@ contract ZkCheatcodesTest is Test {
         vm.etch(TEST_ADDRESS, constantNumberCode);
 
         (bool success, bytes memory output) = TEST_ADDRESS.call(
-            abi.encodeWithSignature(
-                "ten()"
-            )
+            abi.encodeWithSignature("ten()")
         );
         require(success, "ten() call failed");
 
-        (uint8 number) = abi.decode(output, (uint8));
+        uint8 number = abi.decode(output, (uint8));
         require(number == 10, "era etched code incorrect");
     }
+
+    function testRecord() public {
+        vm.record();
+        testSlot = 1;
+        (bytes32[] memory reads, bytes32[] memory writes) = vm.accesses(address(this));
```

Comment:
I would create a new contract to test this. The reason being the `testSlot = 0x1e` can easily change if `ds-test` library adds another field to the `Test` or any of its parent contracts. In a new contract you can be sure it's always the same.

Reply:
For sure, didn't though of that!